src/Serilog.Sinks.AzureTableStorage/Serilog.Sinks.AzureTableStorage.csproj

```diff
@@ -3,27 +3,34 @@
   <PropertyGroup>
     <Description>Serilog event sink that writes to Azure Table Storage over HTTP.</Description>
     <VersionPrefix>5.0.0</VersionPrefix>
+    <!--<VersionSuffix>beta</VersionSuffix>-->
     <Authors>Robert Moore</Authors>
-    <TargetFrameworks>net45;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
Does this change mean we are not going to support net45? I suggest we keep net45 and update netstandard to 2.0 or even 2.1

Reply:
see comments below